scripts_bash/1_data_creation/preprocess_bundles.sh

```diff
@@ -0,0 +1,79 @@
+echo "Processing bundles..."
```

Comment:
This file should not be included now, we should just make a clean one from scratch

Reply:
This is just to save the work. It is not cleaned. (There is not #checked! mark over the function).